tests/model_test.rs

```diff
@@ -130,3 +130,43 @@ fn from_jsonapi_document() {
         }
     }
 }
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+struct NewAuthor {
+    name: String,
+    books: Vec<String>,
+}
+partial_jsonapi_model!(NewAuthor; "authors"; has many books);
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+struct NewBook {
```

Comment:
Should include a test that this model generates included fields and doesn't just smuggle things past seralisation as an attribute.